lib/simctl.ts

```diff
@@ -37,7 +37,16 @@ export class Simctl implements ISimctl {
 	}
 
 	public getAppContainer(deviceId: string, applicationIdentifier: string): IFuture<string> {
-		return this.simctlExec("get_app_container", [deviceId, applicationIdentifier]);
+		return (() => {
+			try {
+				return this.simctlExec("get_app_container", [deviceId, applicationIdentifier]).wait();
+			} catch(e) {
+				if (e.message.indexOf("No such file or directory") > -1) {
```

Comment:
We should get this message only in case when the application is not installed on the simulator.